cloud-service-provider/aws/eks/variables.tf

```diff
@@ -0,0 +1,59 @@
+variable "region" {
+  description = "AWS region"
+  type        = string
+  default     = "eu-west-1"
+}
+
+variable "cluster_name" {
+  description = "EKS cluster name"
+  type        = string
+  default     = null
+}
+
+variable "cluster_version" {
+  description = "EKS cluster version"
+  type        = string
+  default     = "1.31"
+}
+
+variable "instance_types" {
+  description = "EC2 instance types"
+  type        = list
+  default     = ["t3.medium"]
```

Comment:
how does this relate to the Mi7.8xlarge ?

Reply:
this is the default. You override it in the application specific `.tfvars` file like `opea-chatqna.tfvars`. The idea is, that each app has different requirements of the instance they can run.